src/shared/api/baseApi.ts

```
@@ -1,9 +1,45 @@
+  protected constructor() {
+    this.axiosInstance = axios.create({
+      baseURL: process.env.NEXT_PUBLIC_API_URL,
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      withCredentials: false,
+      timeout: 10000,
+    });
+  }
+
+  protected async get<T = any>(
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)